codegen/templates/http_client.tmpl

```diff
@@ -105,20 +105,26 @@ func (c *{{$clientName}}) {{title .Name}}(
 	})
 
 	{{if and (eq .ResponseType "") (eq (len .Exceptions) 0)}}
-	// TODO: log about unexpected body bytes?
-	_, err = res.ReadAll()
-	if err != nil {
-		return respHeaders, err
+	switch res.StatusCode {
+		case {{.OKStatusCode.Code}}:
+			// TODO: log about unexpected body bytes?
+			_, err = res.ReadAll()
```

Comment:
Shouldn't it be considered as an error because response body is expected to contain no bytes?